docs/modules/ROOT/pages/tls.adoc

```diff
@@ -9,13 +9,25 @@ See xref:hazelcast:security:tls-ssl.adoc[TLS/SSL Basics] for more detailed infor
 
 == Prerequisites for TLS
 
-Before you create and configure TLS, you need to create Kubernetes secret:
+Before you create and configure TLS, you need to create Kubernetes secret. Kubernetes secret can be created in two different ways:
+
+If you just want to provide certificates to create the keystore:
 
```

Comment:
Could you provide more information about internal implementation to give some insights to users? So they can create their own solutions.

Reply:
https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/234/commits/29d69be1ddfbc6eef110e906f8959f0d82588f28